docs/mine-hnt/miner-cli/minercli.mdx

```diff
@@ -8,9 +8,73 @@ slug: /mine-hnt/miner-cli/minercli
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
----
+# Introduction
+The `miner` command controls validators and hotspot miners it is also the core software that runs all aspects of the blockchain, like adding txns, and validating blocks while in a consensus group. The CLI is used to troubleshoot and maintain these devices. 
```

Comment:
"The `miner` command controls validators and hotspot miners..." ->
"The `miner` command line interface (CLI) is a tool to administer and interrogate Helium Validators and Hotspots, both of which run the core `miner` software."

Reply:
This was a little unclear hopefully this is what you were going for.